MyOwnORM/MyOwnORM/CustonSqlServerContextOptionsBuilderExtension.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustonSqlServerContextOptionsBuilderExtension
```

Comment:
if it is extension make class and methods static to extend smth like:
...UseSqlServer(this dbContextOptionsBuilder builder, string connection string)

Reply:
Fixed